pkg/config/stepmeta.go

```diff
@@ -105,25 +105,25 @@ type StepOutputs struct {
 // Container defines an execution container
 type Container struct {
 	//ToDo: check dockerOptions, dockerVolumeBind, containerPortMappings, sidecarOptions, sidecarVolumeBind
-	Command         []string    `json:"command"`
-	EnvVars         []EnvVar    `json:"env"`
-	Image           string      `json:"image"`
-	ImagePullPolicy string      `json:"imagePullPolicy"`
-	Name            string      `json:"name"`
-	ReadyCommand    string      `json:"readyCommand"`
-	Shell           string      `json:"shell"`
-	WorkingDir      string      `json:"workingDir"`
-	Conditions      []Condition `json:"conditions,omitempty"`
-	Options         []Option    `json:"options,omitempty"`
-	//VolumeMounts    []VolumeMount `json:"volumeMounts,omitempty"`
+	Command         []string      `json:"command"`
+	EnvVars         []EnvVar      `json:"env"`
+	Image           string        `json:"image"`
+	ImagePullPolicy string        `json:"imagePullPolicy"`
+	Name            string        `json:"name"`
+	ReadyCommand    string        `json:"readyCommand"`
+	Shell           string        `json:"shell"`
+	WorkingDir      string        `json:"workingDir"`
+	Conditions      []Condition   `json:"conditions,omitempty"`
+	Options         []Option      `json:"options,omitempty"`
+	VolumeMounts    []VolumeMount `json:"volumeMounts,omitempty"`
```

Comment:
Maybe we should not go for "VolumeMounts []VolumeMounts", but for "MountPath string". That is what we do for kubernetes anyway (ignoring the name and take the first one). So the docker arguments would be in synch. The user could configure the `mountPath` of an empty and shared volume. WDYT?

Reply:
Well, maybe it DOES make sense to keep the API generic to be able to easily extend the capabilities later.